cmd/placement/options/options.go

```diff
@@ -77,6 +82,12 @@ func New() *Options {
 	opts.Metrics = metrics.DefaultMetricOptions()
 	opts.Metrics.AttachCmdFlags(flag.StringVar, flag.BoolVar)
 
+	// parse env variables before parsing flags, so the flags takes priority over env variables
+	metadataEnable := os.Getenv(envMetadataEnabled)
+	if len(metadataEnable) > 0 {
+		opts.MetadataEnabled = utils.IsTruthy(metadataEnable)
+	}
```

Comment:
An empty value is considered as false, and the default value for `opts.MetadataEnabled` is false.

```suggestion
	opts.MetadataEnabled = utils.IsTruthy(os.Getenv(envMetadataEnabled))
```